util/crs-rules-check/rules-check.py

```diff
@@ -630,6 +653,9 @@ def msg(msg):
         try:
             with open(f, 'r') as inputfile:
                 data = inputfile.read()
+                # modify the content of the file, it it is the "crs-setup.conf.example"
+                if f.startswith("crs-setup.conf.example"):
```

Comment:
Why 'startswith'? Shouldn't be equal to?

Reply:
In practice I'm using this script that I have many different version of `crs-setup.conf.example`, eg. `crs-setup.conf.example.33`, `crs-setup.conf.example.40`, and so on. With `startswith` I can pass that file without modifying of the target file. It's not a big deal - I can replace it, but if we can live with that, I'd keep this.

Reply:
🤷